narwhals/functions.py

```diff
@@ -33,6 +33,112 @@ def concat(
     *,
     how: Literal["horizontal", "vertical"] = "vertical",
 ) -> FrameT:
+    """
+    Concatenate multiple DataFrames, LazyFrames into a single entity.
+
+    Notes:
+        Only horizontal and vertical concatenations are supported.
+
```

Comment:
i think we can just remove this, the arguments section already shows what's supported